versions/3.0.md

```diff
@@ -964,61 +966,71 @@ Describes a set of supported content types. A content object can be used in [req
 
 Each key in the content object is the media-type of the [Content Type Object](#contentTypeObject).
```

Comment:
this should be "media type" ; `media-type` is not a member of the content type object; rather the key is a string which contains a media type.